mibs/yeastar/YEASTAR-PBX-MIB-V4

```diff
@@ -0,0 +1,878 @@
+--
+-- YEASTAR-S-SERIES-V4.my
+-- MIB generated by MG-SOFT Visual MIB Builder Version 6.0  Build 88
+-- Wednesday, April 01, 2020 at 14:28:20
+--
+
+	YEASTAR-S-SERIES-V4 DEFINITIONS ::= BEGIN
```

Comment:
Please rename the MIB file as well, to name it "YEASTAR-S-SERIES-V4" as it is defined inside it.
Thanx